workflow-templates/cloud-defense-dockerimage.yaml

```diff
@@ -0,0 +1,27 @@
+name: CloudDefense Scan for Docker Images
+
+on:
+ pull_request:
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build image
+        run: |
+          docker build -t latest -f docker/ui.Dockerfile .
```

Comment:
This is more oriented toward platform repo setup. Is there a way to either generalize or template out the Dockerfile location and fill it in with a parameter or variable?